service/vector_store_client.hh

```diff
@@ -72,7 +73,7 @@ public:
 
     struct ann_error_visitor {
         sstring operator()(service::vector_store_client::service_error e) const {
-            return fmt::format("Vector Store error: HTTP status {}", e.status);
+            return fmt::format("Vector Store error: HTTP status {}, message: {}", e.status, e.message);
```

Comment:
I see that all variants of the `ann_error` ultimately get converted into a CQL error variant "invalid". The latter represents a validation error, meaning that the user sent a query that is not correct. Not all HTTP error codes will fit to it, as well as some other error variants of `ann_error`. The CQL protocol has more error variants which will fix some `ann_error` variants better, as well as some HTTP error codes, so I suggest improving the conversion routine in the future so that it uses more appropriate CQL error codes.

See the CQL protocol specification: https://github.com/apache/cassandra/blob/70bcaec54d492658cb331b70869215ad64feb63d/doc/native_protocol_v4.spec#L1046

This is a suggestion for further changes, no need to improve anything further in this PR.